elements/rh-range/rh-range.css

```diff
@@ -1,6 +1,8 @@
 /* Adapted from https://codepen.io/ShadowShahriar/pen/zYPPYrQ?editors=0100 */
 :host {
-  display: inline-block;
+  display: flex;
```

Comment:
can we do these on the container?

if we put this stuff on the host, it gives the user a lot of power to screw up the range's internal layout

also

```
$ el
> <input type="range">

$ getComputedStyle(el).getPropertyValue('display')
> "inline-block" 
```